manifests/stage-nv-peer-mem-driver/0050_nv-peer-mem-driver-ds.yaml

```diff
@@ -146,3 +146,10 @@ spec:
                 operator: Gt
                 values:
                   - "0"
+              # cuda driver starting from v465 should include builtin nvidia_peermem module
+              # which should be used instead nv_peer_mem
+              # prevent scheduling nv-peer-mem-driver POD on nodes with cuda driver version >= 465
+              - key: nvidia.com/cuda.driver.major
+                operator: Lt
+                values:
```

Comment:
What value will be in nic-cluster-policy state in case if  version is 465 or above?

Reply:
If all nodes have GPU drivers with version => 465 then from the network-operator point of view this is equal to situation when there are no nodes with GPU in the cluster. In this case, state-NV-Peer will be notReady.

Reply:
I think that misleading to the user and also according to [1] it will keep trying to reconcile. 
can we workaround it that it won't try to reconcile when GPU drivers with version => 465?

[1] - https://github.com/Mellanox/network-operator/blob/e73e49b99ff9d2f0eb92ab95143a9b4890f96d07/pkg/state/state_nv_peer.go#L89-L93

Reply:
@moshe010  IIRC , this will happen if nvidia.com/gpu.present is not set on node.
in this case i believe it will be set and the gpu driver annotation will be >=465.
this will cause the DS to be created but fail on readiness in : https://github.com/Mellanox/network-operator/blob/e73e49b99ff9d2f0eb92ab95143a9b4890f96d07/pkg/state/state_skel.go#L188

Reply:
There is an issue where the order of thing is not really assured after initial deployment however its not related to this PR.